plugin.video.iplayerwww/.gitignore

```diff
@@ -0,0 +1,68 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
do not include in the submission. If using the submitter actions, add to .gitattributes as export-ignore

Reply:
Tried that, but it does not appear to work. What am I doing wrong?

Reply:
Are you using https://github.com/xbmc/action-kodi-addon-submitter ? If not, you need to remove the file from your branch.

Reply:
No, I use standard git.
But I need .gitignore as I have temp files in my local folders that should not get pushed.
What changed here? This used to work for me in helix and krypton.